src/components/Leaderboard.tsx

```diff
@@ -95,10 +96,10 @@ const Leaderboard = ({ content, limit = 100 }: LeaderboardProps) => {
                 <Avatar
                   src={avatarImg}
                   name={avatarAlt}
-                  me={4}
-                  h={10}
-                  w={10}
-                  display={{ base: "none", xs: "block" }}
+                  // This meets the Design System requirement, despite not being used in this instance
+                  href="#"
```

Comment:
Could we align it with the link for the row?
```suggestion
                  href={hasGitHub ? `${GITHUB_URL}${username}` : "#"}
```

Reply:
If not the user gets this ux where the avatar link is upside the linkbox making it a weird experience

https://github.com/user-attachments/assets/129fc872-e7d7-4484-96d5-576e640d700b

Reply:
Hmmm, I wasn't expecting that! I'll go ahead with your suggestion here. 

For future consideration the avatar shouldn't really be hoverable here unless styled based on hover over the rest of the list item, correct?

Reply:
Yes, correct. For this transition, we will leave it as is, but ideally only the row should be the element that gets the hover effect.